src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1090,6 +1091,14 @@ public function resolveServices($value)
             foreach ($value as $k => $v) {
                 $value[$k] = $this->resolveServices($v);
             }
+        } elseif ($value instanceof LocatorArgument) {
+            $services = array();
+            foreach ($value->getValues() as $k => $v) {
+                $services[$k] = function () use ($k, $v) {
+                    return $v instanceof Reference ? $this->get((string) $v, $v->getInvalidBehavior()) : $v;
```

Comment:
not need for `$k` in the `use`
`return $this->resolveServices($parameterBag->unescapeValue($parameterBag->resolveValue($v)));`